skein/model.py

```diff
@@ -472,20 +472,24 @@ class ApplicationSpec(Specification):
         The queue to submit to. Defaults to the default queue.
     tags : set, optional
         A set of strings to use as tags for this application.
+    name_nodes : list, optional
+        A list of namenode URIs to acquire delegation tokens for.
```

Comment:
I haven't used ViewFs before. Is this a normal way of handling this (what do other tools do)? Is there a way to introspect what namenode uris are needed from configuration somewhere?

Reply:
Thanks for the information.

Given Spark deprecated `spark.yarn.access.namenodes` in favor of `spark.yarn.access.hadoopFileSystems`, perhaps we should call this field something similar as well. A few ideas (no strong opinions):

- `hadoop_file_systems`
- `additional_file_systems` (since the default one is always acquired)
- `file_systems`

Reply:
`additional_file_systems` look good, will amend the PR.